### ISSUE TITLE
Use SVG for table header sort icons

### ISSUE THREAD

**Issue by NormPlum** [backdrop/backdrop-issues#6779]
## Description of the need

Table header sort icons are still PNG images. SVGs would allow better styling especially on different colour backgrounds.

## Proposed solution

Change PNG icons to SVG.

## Alternatives that have been considered

Using CSS was mentioned as an option in https://github.com/backdrop/backdrop-issues/issues/6548 but I don't know how that works.

**Comment by indigoxela** [comment]
@NormPlum many thanks for providing this PR. :pray: 

Oops, Seven - our default admin theme - overrides that theme function.
I wonder, if it makes sense to fix it also in Seven: https://github.com/backdrop/backdrop/blob/1.x/core/themes/seven/template.php#L91

Or possibly drop that override in Seven, as the icon should show up in the right color now, anyway. Didn't test that, though - that's only based on assumption.

On the other hand changes in core themes are usually a difficult topic and I don't want to talk you into something... :grimacing:

**Comment by stpaultim** [comment]
@NormPlum YES, these two issues very much overlap and maybe should be combined. Personally, I think it would make sense to merge this issue into the older issue and try to fix them both together in that other issue. 

I think it would be silly to fix that one without usign SVG icons.

**Comment by indigoxela** [comment]
I took a look now, by setting Basis to admin theme and going to admin/content.

Love it! :+1: 

Besides the fact that the SVG also looks nice and crisp when rescaling the page in the browser, this also fixes an annoying contrast problem.

To compare:

![old-img-low-contrast](https://github.com/user-attachments/assets/184d4015-7b0b-4382-bfc1-925daa4333b7)

^^ hard to recognize and doesn't even appear to be connected to the table header label.

![new-img-better-contrast](https://github.com/user-attachments/assets/f1f10999-de32-4587-87e8-9c0d1a429ee3)

^^ Improves visual connection and contrast.

Works for me. :smiley:

**Comment by indigoxela** [comment]
Because of the contrast fix, I went with "milestone candidate bug" - it fixes an A11Y issue. But we can also set it to "minor".

**Comment by herbdool** [comment]
I think this is ready. I wouldn't tie to the discussion of changing the icon in Seven. The only thing that we could include -- though not necessary -- is to change the existing Seven icons to their svg versions.

**Comment by indigoxela** [comment]
> we could include -- though not necessary -- is to change the existing Seven icons to their svg versions.

Re sort icon in Seven we do have an open issue, so let's discuss that over [there](https://github.com/backdrop/backdrop-issues/issues/5865).
This PR here seems good to go as is. 😉 @NormPlum many thanks for your patience. 🙏

**Comment by quicksketch** [comment]
Thank you @NormPlum, @indigoxela, @herbdool, and @stpaultim! I agree this is good-to-go as-is. Thank you for your patience! I merged https://github.com/backdrop/backdrop/pull/4932 into 1.x and 1.32.x.
